### PR TITLE
Added new option to manage logo validation multiplier

### PR DESCRIPTION
**Pull request by vsvp21**
It is actually works fine with 2 * logoSize, camera scans QR easily and without any problems

### CODE REVIEW

**Comment by yeqown** [comment]
What about converting `2 * logoSize` as `N * logoSize`, N can be controlled easilythe  by user. And further, `validLogoImage` can be blocked by user @vsvp21

**Comment by vsvp21** [comment]
Added new option to control this validation @yeqown

File: writer/standard/image_option.go [hunk 1]
```diff
@@ -2,10 +2,9 @@ package standard
 
 import (
 	"fmt"
+	"github.com/yeqown/go-qrcode/v2"
```
Comment (yeqown) [inline writer/standard/image_option.go:5]: Keeps imports in order of [std, library itself, third party]

File: writer/standard/writer.go [hunk 2]
```diff
@@ -2,13 +2,13 @@ package standard
 
 import (
 	"fmt"
+	"github.com/yeqown/go-qrcode/v2"
```
Comment (yeqown) [inline writer/standard/writer.go:5]: Keeps imports sequence as above

**Review by yeqown — CHANGES_REQUESTED** [review]

File: writer/standard/image_option.go (hunk 1 above)
Comment (vsvp21) [reply writer/standard/image_option.go]: @yeqown fixed

File: writer/standard/writer.go (hunk 2 above)
Comment (vsvp21) [reply writer/standard/writer.go]: @yeqown fixed

**Review by yeqown — APPROVED** [review]

**Pull request merged**